includes/data-port/class-sensei-data-port-utilities.php

```diff
@@ -40,4 +40,123 @@ public static function create_or_update_user( $username, $email = '' ) {
 
 		return $user->ID;
 	}
+
+	/**
+	 * Cet a term based on human readable string and create it if needed. If the taxonomy is hierarchical,
+	 * this method processes that as well and returns the \WP_Term object for the last in their hierarchy.
+	 *
+	 * @param string $term_name_path  Term name with optional hierarchy path, separated by " > ".
+	 * @param string $taxonomy_name   Name of the taxonomy.
+	 * @param int    $teacher_user_id User ID for the teacher (only needed for modules).
+	 *
+	 * @return WP_Term|false
+	 */
+	public static function get_term( $term_name_path, $taxonomy_name, $teacher_user_id = null ) {
+		$taxonomy = get_taxonomy( $taxonomy_name );
+		if ( ! $taxonomy ) {
+			return false;
+		}
+
+		if ( $taxonomy->hierarchical ) {
+			$term_path = preg_split( '/ ?\> ?/', $term_name_path );
```

Comment:
I think that `>` does not need escaping.

Reply:
I tend to escape everything, but you're right, no need c697d758d8fb5998237b970657d044b8ce82c712